week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,17 +1,18 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+return "Hello, my name is " + name + " and I am " + age +  " years old";
+}
 
 function getRemainder(a, b) {
-    remainder = a %% b;
+     var remainder = a % b;
 
     // Use string interpolation here
-    return "The remainder is %{remainder}"
+    return "The remainder is " + remainder
 }
```

Comment:
You should use the string interpolation here. Search what is a string interpolation and fix this part of the function.